src/Build/Resources/Strings.resx

```diff
@@ -1306,6 +1306,9 @@
   <data name="SDKResolverFailed" xml:space="preserve">
     <value>The SDK resolver "{0}" failed while attempting to resolve the SDK "{1}". Exception: "{2}"</value>
   </data>
+  <data name="FailedToResolveSDK" xml:space="preserve">
+    <value>No SDK resolver succeeded in resolving SDK "{0}". Their error messages are printed below. Only one of the following indicates a real failure:</value>
```

Comment:
@baronfel this could use some wordsmithing, got opinions?

```suggestion
    <value>No SDK resolver could resolve SDK "{0}". Their error messages are printed below. Failures are expected; only one of the following indicates a real failure:</value>
```

Reply:
As a naive user, I find this message a bit confusing. I assume what it means is -- this is something like the list of loader errors -- you would expect errors as it "looks around", but in this case, they weren't followed by a successful find.

If that's the case, I think I'd expect something like this would actually be clearer for me.
`<value>None of the SDK resolvers were able to resolve SDK "{0}". Each resolver's error message is shown below.</value>`

Reply:
My other thought is -- is it implicit that the errors are being emitted in the order they occurred? I guess it probably is, but if not it could mention that.

Reply:
Strong agree that we should clarify that order is important.

Reply:
Proposed wording:
None of the SDK resolvers were able to resolve SDK "{0}". Their error messages are reproduced below in the order they occurred. Only one SDK resolver is expected to be able to resolve "{0}", so only one of the failures is a real failure.

Thoughts? As far as error codes, my code kinda looks like it's removing error codes, but in reality, it leaves them as long as they're part of the message. MSBuild directly controls the DefaultSDKResolver, but that's it; the other SDK resolvers will have to introduce their own error codes separately.

Reply:
An alternative framing to "of these errors only one is real" might be to present it as just an activity trace. When we look at dll load failures, or fusion logs, for example, we don't consider any of the output to be errors. It just says what happened. So eg 

`<value>Could not resolve SDK "{0}". Detailed trace from each SDK resolver follows below.</value>`

or something.